src/main/java/duke/data/exception/IllegalValueException.java

```diff
@@ -0,0 +1,13 @@
+package src.main.java.duke.data.exception;
+
+/**
+ * Signals that some given data does not fulfill some constraints.
+ */
+public class IllegalValueException extends DukeException {
```

Comment:
Perhaps for classes like these that extend off dukeException, we could add additional information about the error itself as compared to having the very generic dukeException error.
This could make the addition of a IllegalValueException have some additional value add.